### ISSUE TITLE
clamav-unofficial-sigs.pid: Permission denied; Removing unused file: /var/lib/clamav/spam_marketing.ndb

### ISSUE THREAD

**Issue by RobbieTheK** [extremeshok/clamav-unofficial-sigs#342]
I'm seeing these errors after upgrading on Fedora 33:

`Sanesecurity GPG Signature test FAILED on winnow_malware.yara database - SKIPPING`

`Removing unused file: /var/lib/clamav/spam_marketing.ndb`

```
/usr/local/sbin/clamav-unofficial-sigs.sh: line 75:
/var/lib/clamav-unofficial-sigs/pid/clamav-unofficial-sigs.pid: Permission denied
================================================================================================
ERROR: Could not create PID file:
/var/lib/clamav-unofficial-sigs/pid/clamav-unofficial-sigs.pid
```

**Comment by extremeshok** [comment]
have you run the the script after updating as the root user
```
/usr/local/sbin/clamav-unofficial-sigs.sh --force
```

**Comment by RobbieTheK** [comment]
> have you run the the script after updating as the root user
> 
> ```
> /usr/local/sbin/clamav-unofficial-sigs.sh --force
> ```

Yes that's where I get:

```
Testing updated Sanesecurity database file: winnow_malware.yara
Sanesecurity GPG Signature test FAILED on winnow_malware.yara database - SKIPPING
```

I ran `touch /var/lib/clamav-unofficial-sigs/pid/clamav-unofficial-sigs.pid` to suppress the PID error.

**Comment by archifishaldev** [comment]
I get the 'Removing unused file: /var/lib/clamav/spam_marketing.ndb' every couple of hours or so (cronjob email) - I'd originally assumed it was failing to remove it for some reason, but actually there is no such file.

**Comment by extremeshok** [comment]
not sure about the spam_marketing.ndb error.

I would need your user.conf and your os.conf to debug it.

**Comment by extremeshok** [comment]
Done lots of testing with 7.2.1 and I cannot recreate the error.

**Comment by RobbieTheK** [comment]
Just updated to 7.2.1 getting this on Fedora 32:
```
/usr/local/sbin/clamav-unofficial-sigs.sh: line 75:
/var/lib/clamav-unofficial-sigs/pid/clamav-unofficial-sigs.pid: Permission denied
================================================================================================
ERROR: Could not create PID file:
/var/lib/clamav-unofficial-sigs/pid/clamav-unofficial-sigs.pid
================================================================================================
```

**Comment by archifishaldev** [comment]
@RobbieTheK Does the above file exist when the script isn't running? I'm guessing it shouldn't be present if it is currently (certainly there's nothing there here in general)

**Comment by RobbieTheK** [comment]
LMK if I should open another issue but after upgrading to 7.2.2 and running `clamav-unofficial-sigs.sh --force`
```
Testing updated Sanesecurity database file: winnow_malware.yara
Sanesecurity GPG Signature test FAILED on winnow_malware.yara database - SKIPPING

```

**Comment by extremeshok** [comment]
fixed : 7.2.4
